cmd/genesis/account.go

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/genesis/model"
+)
+
+func init() {
+	accountCmd.PersistentFlags().String("server", defaultLocalServerAPI, "The provisioning server whose API will be queried.")
+	accountCmd.PersistentFlags().Bool("dry-run", false, "When set to true, only print the API request without sending it.")
+
+	accountCreateCmd.Flags().String("service-catalog-product", "", "The service catalog product id to provision a new account")
+	accountCreateCmd.Flags().String("provider", "aws", "Cloud provider hosting the account.")
+	accountCreateCmd.Flags().Bool("provision", false, "When set to true provision an account after creation.")
+
+	accountCreateCmd.MarkFlagRequired("service-catalog-product")
+
+	accountProvisionCmd.Flags().String("account", "", "The id of the account to be deleted.")
+	accountProvisionCmd.MarkFlagRequired("	account")
+
+	accountDeleteCmd.Flags().String("account", "", "The id of the account to be deleted.")
+	accountDeleteCmd.MarkFlagRequired("	account")
```

Comment:
I assume the gaps are placeholders, right?

Reply:
Your assumption is wrong. It a typo, thank you :P